src/Symfony/Component/DependencyInjection/ContainerBuilder.php

```diff
@@ -279,6 +285,36 @@ public function addClassResource(\ReflectionClass $class)
     }
 
     /**
+     * Retrieves the requested reflection class and registers it for resource tracking.
+     *
+     * @param string $class
+     *
+     * @return \ReflectionClass|false
```

Comment:
Can we use ``\ReflectionClass|null`` instead ? It would be cleaner (PHP 7.1 has nullable types, not falsable types)